### ISSUE TITLE
https://pytorch.org/docs/stable/_modules/torch/nn/modules/loss.html

### ISSUE THREAD

**Issue by voidiker66** [pytorch/pytorch.github.io#52]
This used to have docs regarding the loss functions. Where did they go?

**Comment by soumith** [comment]
it's still there: https://pytorch.org/docs/stable/_modules/torch/nn/modules/loss.html

other docs for loss functions are here https://pytorch.org/docs/stable/nn.html#loss-functions and here https://pytorch.org/docs/stable/nn.html#id50
